kratos/containers/global_pointers_vector.h

```diff
@@ -199,21 +199,6 @@ class GlobalPointersVector final
         return mData[i];
     }
 
-    /**
-     * @brief Equality comparison operator to check if two GlobalPointersVector objects are equal.
-     * @details This function checks if the sizes are equal and then compares the elements for equality
-     * using the EqualKeyTo() function.
-     * @param r The GlobalPointersVector to compare with.
-     * @return True if the containers are equal, false otherwise.
-     */
-    bool operator==(const GlobalPointersVector& r) const // nothrow
```

Comment:
hi @matekelemen it looks to me that the comparison would make sense (provided it compiles of couse)
wouldn't it be better to implement the EqualKeyTo rather than removing the method?

Reply:
Well, I don't see a use case for comparing the contents of entire containers that wouldn't be possible with `std::equal`, but in any case, I implemented the function then.